Client/Frontend/Browser/Toolbars/ToolbarProtocols.swift

```diff
@@ -0,0 +1,35 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+protocol ToolbarProtocol: class {
```

Comment:
No changes here besides actions for the new menu and removing params from `tabToolbarDidPressShare`